magma/magma
# feat(orc8r): Add converged core config flags

## Summary

Changes for Enable / Disable 5G SA. Changes on Orc8r side. Removed "enable_converged_core" from mme.yml file as it takes precedence over data updated from Orchestrator.
Key Added : "enable_converged_core"
Request Type : GET/PUT
API url : "/lte/{network_id}/cellular/epc"


## Test Plan

UERANsim simulator tested below things :
1. Update in config Files. (/var/opt/magma/configs/gateway.mconfig, /var/opt/magma/tmp/mme.conf)
2. Check the result on /var/log/mme.log
3. UE registration procedure on "magma-5G" branch
4. NGSetupRequest - to establish SCTP connection on "magma-5G" branch 

## Additional Information


Signed-off-by: Prabhat Mishra <prabhat.mishra@wavelabs.ai>
